assets/js/base/hooks/use-store-cart-item-quantity.js

```diff
@@ -13,25 +11,28 @@ import { useDebounce } from 'use-debounce';
  */
 import { useStoreCart } from './use-store-cart';
 
+/**
+ * @typedef {import('@woocommerce/type-defs/hooks').StoreCartItemQuantity} StoreCartItemQuantity
+ * @typedef {import('@woocommerce/type-defs/cart').CartItem} CartItem
+ */
+
 /**
  * This is a custom hook for loading the Store API /cart/ endpoint and
  * actions for removing or changing item quantity.
  *
  * @see https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/master/src/RestApi/StoreApi
  *
  * @param {string} cartItemKey Key for a cart item.
- * @return {StoreCartItem} An object exposing data and actions relating to cart items.
+ * @return {StoreCartItemQuantity} An object exposing data and actions relating to cart items.
```

Comment:
Nice, I think it's clearer to focus this on quantity / actions (including remove action) :)